README.md

```diff
@@ -14,7 +14,8 @@ Searvey aims to provide the following functionality:
 - On demand data retrieval from multiple sources that currently include:
 
     - U.S. Center for Operational Oceanographic Products and Services (CO-OPS)
-    - Flanders Marine Institute (VLIZ); Intergovernmental Oceanographic Commission (IOC)
+    - Flanders Marine Institute (VLIZ)
+    - Intergovernmental Oceanographic Commission (IOC)
```

Comment:
My understanding is that IOC is managed by VLIZ or something like that. @brey? What is the proper way to handle this?

Reply:
VLIZ does the technical implementation for IOC. Listing both like this is fine. 
Referring to the DOI is even better [10.14284/482](https://doi.org/10.14284/482)

Reply:
Sorry I misunderstood why they were listed on the same line. Should I just revert it to how it was?

Reply:
reverting is consistent with the reference from @bart-v